README.md

```diff
@@ -85,4 +85,64 @@ Finally, let's start Server.
   server.exe
   ```
 # Quickstart
-// TODO
\ No newline at end of file
+**Step 1: Downloading a Model**
+
+```bash
+mkdir model && cd model
+wget -O llama-2-7b-model.gguf https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q5_K_M.gguf?download=true
+```
+
+**Step 2: Start server**
```

Comment:
Can you add a link back to the `## Start process` section for a better instruction.

Reply:
Will update in https://github.com/janhq/cortex.llamacpp/pull/23
This PR has a commit of https://github.com/janhq/cortex.llamacpp/pull/23